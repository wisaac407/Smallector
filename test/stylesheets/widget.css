/*
 * commands: split
 */

.widget {
  background-color: #FF0000;
}

.widget-header {
  background-color: #FF0000;
}

.widget-title {
  background-color: #FF0000;
}

.widget-body {
  background-color: #FF0000;
}

.widget-footer {
  background-color: #FF0000;
}