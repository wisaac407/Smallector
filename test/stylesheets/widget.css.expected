/*
 * commands: split
 */

.a {
  background-color: #FF0000;
}

.a-e {
  background-color: #FF0000;
}

.a-d {
  background-color: #FF0000;
}

.a-c {
  background-color: #FF0000;
}

.a-b {
  background-color: #FF0000;
}